src/modules/rover_differential/module.yaml

```diff
@@ -156,6 +158,21 @@ parameters:
         decimal: 2
         default: 90
 
+      RD_MAX_YAW_ACCEL:
+        description:
+          short: Maximum allowed yaw acceleration for the rover
+          long: |
+            This parameter is used to cap desired yaw acceleration. This is used to adjust incoming yaw rate setpoints
+            to a feasible yaw rate setpoint based on the physical limitation on how fast the yaw rate can change.
+            This leads to a smooth setpoint trajectory for the closed loop yaw rate controller to track.
```

Comment:
Explain what it being set to 0 means.

Reply:
Changed default to -1 which disables this feature and added "Set to -1 to disable" to the description.
0 also disables it, but I'll use the set to -1 to disable convention here.